cluster-autoscaler/cloudprovider/cherryservers/README.md

```diff
@@ -29,6 +29,7 @@ In the above file you can modify the following fields:
 | cluster-autoscaler-cloud-config | Global/cloudinit        | The base64 encoded user data submitted when provisioning servers. In the example file, the default value has been tested with Ubuntu 18.04 to install Docker & kubelet and then to bootstrap the node into the cluster using kubeadm. The kubeadm, kubelet, kubectl are pinned to version 1.17.4. For a different base OS or bootstrap method, this needs to be customized accordingly. It will use go templates to inject runtime information; see below.|
 | cluster-autoscaler-cloud-config | Global/reservation      | The values "require" or "prefer" will request the next available hardware reservation for new servers in selected region & plan. If no hardware reservations match, "require" will trigger a failure, while "prefer" will launch on-demand servers instead (default: none)  |
 | cluster-autoscaler-cloud-config | Global/hostname-pattern | The pattern for the names of new Cherry Servers servers (default: "k8s-{{.ClusterName}}-{{.NodeGroup}}-{{.RandString8}}" )                  |
+| cluster-autoscaler-cloud-config | Global/os-partition-size | The OS partition size in gigabytes for new nodes in the nodepoll (eg: `60`)                 |
```

Comment:
This should be `nodepool` instead of `nodepoll`.

Reply:
Also, should describe the default.

Reply:
Thanks, made a fix.